### PR TITLE
[branch2]make new branch and new folder and new file

### PR DESCRIPTION
**Pull request by fatemeFeyzabadi68**
1- make new folder
2- make new file

### CODE REVIEW

File: F3/text3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this is new file
```
Comment (fatemeFeyzabadi68) [inline F3/text3.txt:1]: 🤦‍♀️

Comment (fatemeFeyzabadi68) [reply F3/text3.txt:1]: why?

Comment (feyzekcc) [reply F3/text3.txt:1]: is good

Comment (feyzekcc) [reply F3/text3.txt:1]: awsome

Comment (feyzekcc) [reply F3/text3.txt:1]: finish

**Review by feyzekcc — APPROVED** [review]

**Pull request merged**